cmd/operator-sdk/bundle/create.go

```diff
@@ -46,59 +46,76 @@ func newCreateCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "create",
 		Short: "Create an operator bundle image",
-		Long: `The 'operator-sdk bundle create' command will build an operator
-bundle image containing operator metadata and manifests, tagged with the
-provided image tag.
+		Long: `
+The 'operator-sdk bundle create' command will build an operator bundle image containing operator metadata and 
+manifests, tagged with the provided image tag. 
+		
+To write all files required to build a bundle image without building the image, set '--generate-only=true'. 
+		
+A bundle Dockerfile, bundle metadata, and a 'manifests/' directory containing your bundle manifests will 
+be written if '--generate-only=true'. Note that, if the 'manifests/' do not exist, this command will 
+create the directory which is expected with the CRD's resources from the '/deploy/crds/' directory and generate 
+a CSV file that can be applied to the cluster using standard tooling like kubectl and defines its 
+integration with OLM. 
+
+Also, the /metadata directory will be created with resources that are used to store supporting 
+metadata associated with the operator.
+		
+Notes:
+* The image is not runnable.
+* More info on Operator Bundle images and metadata format see: 
+  https://github.com/operator-framework/operator-registry#manifest-format 
+`,
+		Example: `
+The following invocation will build a memcached-operator bundle metadata locally which is useful 
+if you want to build an operator's bundle image manually, modify metadata before building an image, 
+or want to generate a 'manifests/' directory containing your operator manifests for compatibility
+with other operator tooling.
 
-To write all files required to build a bundle image without building the
```

Comment:
We should not add Examples in the Lng Description. 
For it, we should use Examples.